<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
	<meta name="google-site-verification" content="MoR9-zmnPM1LMkTPIjp1GhWG192L2Yv8wTwtZGUqXVM" />
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
	<meta name="google-site-verification" content="hRWbpxG41y_dQCgyn9ixgd180Bz3c8Z8zBkN8ItnNB8" />
	<link rel="stylesheet" href="./datafiles/jemdoc.css" type="text/css">
	<title>Yucheng Yang</title>
	<script type="text/javascript" async="" src="./datafiles/ga.js"></script><script type="text/javascript" async="" src="./datafiles/map.js"></script><script type="text/javascript" async="" src="./datafiles/ga(1).js"></script><script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-39824124-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

	</script><div><style>*{font-family:undefined,sans-serif!important;}</style></div>
	<script type="text/javascript" src="./datafiles/jquery.min.js"></script><script type="text/javascript" src="./datafiles/jquery.min(1).js"></script>
</head>
<body data-feedly-mini="yes">

<div id="layout-content" style="margin-top:25px">

<!-- <div style="text-align:right;">
<b><font color="gray">Home</font></b>&nbsp;&nbsp;|&nbsp; <b><a href="team.html">Team</a></b>&nbsp;&nbsp;|&nbsp; <b><a href="research.html">Research</a></b>&nbsp;&nbsp;|&nbsp; <b><a href="publications.html">Publications</a></b>
</div> -->

<br><br>

<table>
	<tbody>
		<tr>
			<td valign="bottom">
				<div id="toptitle">
					<h1><font size="6" color="#062161">Yucheng T. Yang</font> &nbsp; </h1><h1>
				</h1></div>

				<p> 
					<font size="3">
						<p><a href="https://istbi.fudan.edu.cn/"> Institute of Science and Technology for Brain-inspired Intelligence </a></p>
						<p><a href="https://cnbi.fudan.edu.cn/"> MOE Key Laboratory of Computational Neuroscience and Brain-Inspired Intelligence </a></p>
						<p><a href="https://www.fudan.edu.cn/"> Fudan University </a></p>
						<p>Email: yangyy [at] fudan.edu.cn</p>
						<p>Office: Room 2320, East Main Building of Guanghua Tower</p>
						<p>Address: Fudan University, 220 Handan Rd., Shanghai 200433, China</p>
				</font> </p>
			</td>

		</tr>
	</tbody>
</table>

<br><br>
	
<h2><font color="#0371BB">About</font></h2>
	
<p>I received my Ph.D. degree in Bioinformatics from <strong>Tsinghua University</strong> in 2016 (Advisor: Prof. <a href="https://www.ncrnalab.org/home/">Zhi John Lu</a>), 
	and M.S. degree in statistics from <strong>University of California, Los Angeles</strong> in 2017 (Advisor: Prof. <a href="http://jsb.ucla.edu/">Jingyi Jessica Li</a>).
	Then I did my postdoc training at <strong>Yale University</strong> (Advisor: Prof. <a href="http://www.gersteinlab.org/">Mark B. Gerstein</a>).
	Currently, I'm an Assistant Professor at the Institute of Science and Technology for Brain-inspired Intelligence (ISTBI), <strong>Fudan University</strong>.</p>

<br><br>


<h2><font color="#0371BB">Research</font></h2>
	
<p>I work in the area of bioinformatics and computational genomics.
	My research focuses on applying computational and statistical methods to uncover the underlying principles of <strong>gene regulatory networks</strong>, 
	and improve our understanding on the <strong>genotype-phenotype relationship</strong> in complex biological systems and/or disease context.</p>
<p>I am particularly interested in interpreting the functions of <strong>non-coding genome</strong>, 
	and how the <strong>genetic variation</strong> in the non-coding regions can result in <strong>brain disorders</strong> and other human diseases.</p>

<br><br>

	
<h2><font color="#0371BB">Publications</font></h2>
	
<strong>Full publication list:</strong> <a href="https://scholar.google.com/citations?user=zRH2o9UAAAAJ&hl=en">Google Scholar</a> ; <a href="https://www.researchgate.net/profile/Yucheng-Yang-11">ResearchGate</a><br>
	<br>
<strong>author*</strong>: co-first author, equal contribution<br>
	<br>
<strong><u>author</u></strong>: corresponding author<br>
	<br><br>

<center><h3><font color="#000000"><strong>-- Preprint --</strong></font></h3></center>
	<strong> Multi-tissue integrative analysis of personal epigenomes </strong><br>
	J Rozowsky*, J Drenkow*, <strong>YT Yang*</strong>, G Gursoy*, T Galeev*, B Borsari*, CB Epstein*, K Xiong*, J Xu*, J Gao*, K Yu, A Berthel, Z Chen, F Navarro, J Liu, MS Sun, J Wright, J Chang, CJF Cameron, N Shoresh, E Gaskell, J Adrian, S Aganezov, G Balderrama-Gutierrez, S Banskota, GB Coronoa, S Chee, SB Chhetri, GCC Martins, C Danyko, CA Davis, D Farid, NP Farrell, I Gabdank, Y Gofin, DU Gorkin, M Gu, V Hecht, BC Hitz, R Issner, M Kirsche, X Kong, BR Lam, S Li, B Li, T Li, X Li, KZ Lin, R Luo, M Mackiewicz, JE Moore, J Mudge, N Nelson, C Nusbaum, I Popov, HE Pratt, Y Qiu, S Ramakrishnan, J Raymond, L Salichos, A Scavelli, JM Schreiber, FJ Sedlazeck, LH See, RM Sherman, X Shi, M Shi, CA Sloan, JS Strattan, Z Tan, FY Tanaka, A Vlasova, J Wang, J Werner, B Williams, M Xu, C Yan, L Yu, C Zaleski, J Zhang, JM Cherry, EM Mendenhall, WS Noble, Z Weng, ME Levine, A Dobin, B Wold, A Mortazavi, B Ren, J Gillis, RM Myers, MP Snyder, J Choudhary, A Milosavljevic, <u>MC Schatz</u>, <u>R Guigó</u>, <u>BE Bernstein</u>, <u>TR Gingeras</u>, <u>M Gerstein</u>.
	(2021) <strong><em>bioRxiv</em></strong>.<br>
	<a href="https://www.researchgate.net/publication/351196342_Multi-tissue_integrative_analysis_of_personal_epigenomes"> Full Text </a>
	<br><br>
	
	<strong> A community-driven roadmap to advance Ribo-seq open reading frame research </strong><br>
	<u>JM Mudge</u>*, <u>J Ruiz-Orera</u>*, <u>JR Prensner</u>*, MA Brunet, JM Gonzalez, M Magrane, T Martinez, JF Schulz, <strong>YT Yang</strong>, MM Alba, PV Baranov, A Bazzini, E Bruford, MJ Martin, A-R Carvunis, J Chen, JP Couso, P Flicek, A Frankish, M Gerstein, N Hubner, NT Ingolia, G Menschaert, U Ohler, X Roucou, A Saghatelian, J Weissman, <u>S van Heesch</u>.
	(2021) <strong><em>bioRxiv</em></strong>.<br>
	<a href="https://www.researchgate.net/publication/352305669_A_community-driven_roadmap_to_advance_research_on_translated_open_reading_frames_detected_by_Ribo-seq"> Full Text </a>
	<br><br>

	
<center><h3><font color="#000000"><strong>-- 2022 --</strong></font></h3></center>
	<strong> POSTAR3: an updated platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins </strong><br>
	W Zhao*, S Zhang*, Y Zhu, X Xi, P Bao, Z Ma, TH Kapral, S Chen, B Zagrovic, <strong><u>YT Yang</u></strong>, <u>ZJ Lu</u>.
	(2021) <strong><em>Nucleic Acids Research</em></strong> 50(D1):D287-D294.<br>
	<a href="https://www.researchgate.net/publication/353967531_POSTAR3_an_updated_platform_for_exploring_post-transcriptional_regulation_coordinated_by_RNA-binding_proteins"> Full Text </a>
	<br><br>
	
<center><h3><font color="#000000"><strong>-- 2021 --</strong></font></h3></center>
	<strong> Do Students With Different Majors Have Different Personality Traits? Evidence From Two Chinese Agricultural Universities </strong><br>
	X Wen*, Y Zhao*, <strong>YT Yang</strong>, S Wang, <u>X Cao</u>.
	(2021) <strong><em>Frontiers in Psychology</em></strong> 12:641333.<br>
	<a href="https://www.researchgate.net/publication/351197709_Do_Students_With_Different_Majors_Have_Different_Personality_Traits_Evidence_From_Two_Chinese_Agricultural_Universities"> Full Text </a>
	<br><br>
	
	<strong> Predicting dynamic cellular protein-RNA interactions by deep learning using in vivo RNA structure </strong><br>
	L Sun*, K Xu*, W Huang*, <strong>YT Yang*</strong>, P Li, L Tang, T Xiong, <u>QC Zhang</u>.
	(2021) <strong><em>Cell Research</em></strong> 31(5):495-516.<br>
	<a href="https://www.researchgate.net/publication/349566477_Predicting_dynamic_cellular_protein-RNA_interactions_by_deep_learning_using_in_vivo_RNA_structures"> Full Text </a>
	<br><br>
	
	<strong> GENCODE 2021 </strong><br>
	A Frankish, M Diekhans, I Jungreis, J Lagarde, JE Loveland, JM Mudge, C Sisu, JC Wright, J Armstrong, I Barnes, A Berry, A Bignell, C Boix, SC Sala, F Cunningham, TD Domenico, S Donaldson, IT Fiddes, CG Girón, JM Gonzalez, T Grego, M Hardy, T Hourlier, KL Howe, T Hunt, OG Izuogu, R Johnson, FJ Martin, L Martínez, S Mohanan, P Muir, FCP Navarro, A Parker, B Pei, F Pozo, FC Riera, M Ruffier, BM Schmitt, E Stapleton, MM Suner, I Sycheva, B Uszczynska-Ratajczak, MY Wolf, J Xu, <strong>YT Yang</strong>, A Yates, D Zerbino, Y Zhang, JS Choudhary, M Gerstein, R Guigó, TJP Hubbard, M Kellis, B Paten, ML Tress, <u>P Flicek</u>.
	(2021) <strong><em>Nucleic Acids Research</em></strong> 49(D1):D916-D923.<br>
	<a href="https://www.researchgate.net/publication/347782667_GENCODE_2021"> Full Text </a>
	<br><br>
	
<center><h3><font color="#000000"><strong>-- 2020 --</strong></font></h3></center>
	<strong> Predicting changes in protein thermodynamic stability upon point mutation with deep 3D convolutional neural networks </strong><br>
	B Li, <strong>YT Yang</strong>, <u>JA Capra</u>, <u>MB Gerstein</u>.
	(2020) <strong><em>PLoS Computational Biology</em></strong> 16(11):e1008291.<br>
	<a href="https://www.researchgate.net/publication/347653897_Predicting_changes_in_protein_thermodynamic_stability_upon_point_mutation_with_deep_3D_convolutional_neural_networks"> Full Text </a>
	<br><br>
	
	<strong> An integrative ENCODE resource for cancer genomics </strong><br>
	J Zhang*, D Lee*, V Dhiman*, P Jiang*, J Xu*, P McGillivray*, H Yang*, J Liu, W Meyerson, D Clarke, M Gu, S Li, S Lou, J Xu, L Lochovsky, M Ung, L Ma, S Yu, Q Cao, A Harmanci, KK Yan, A Sethi, G Gursoy, MR Schoenberg, J Rozowsky, J Warrell, P Emani, <strong>YT Yang</strong>, T Galeev, X Kong, S Liu, X Li, J Krishnan, Y Feng, JC Rivera-Mulia, J Adrian, JR Broach, M Bolt, J Moran, D Fitzgerald, V Dileep, T Liu, S Mei, T Sasaki, C Trevilla-Garcia, S Wang, Y Wang, C Zang, D Wang, R Klein, M Snyder, DM Gilbert, K Yip, C Cheng, <u>F Yue</u>, <u>XS Liu</u>, <u>K White</u>, <u>M Gerstein</u>.
	(2020) <strong><em>Nature Communications</em></strong> 11(1):3696.<br>
	<a href="https://www.researchgate.net/publication/343300283_An_integrative_ENCODE_resource_for_cancer_genomics"> Full Text </a>
	<br><br>
	
<center><h3><font color="#000000"><strong>-- 2019 --</strong></font></h3></center>
	<strong> Building a Hybrid Physical-Statistical Classifier for Predicting the Effect of Variants Related to Protein-Drug Interactions </strong><br>
	B Wang*, C Yan*, S Lou, P Emani, B Li, M Xu, X Kong, W Meyerson, <strong>YT Yang</strong>, D Lee, <u>M Gerstein</u>.
	(2019) <strong><em>Structure</em></strong> 27(9):1469-1481.<br>
	<a href="https://www.cell.com/structure/fulltext/S0969-2126(19)30200-X"> Full Text </a>
	<br><br>
	
	<strong> POSTAR2: deciphering the post-transcriptional regulatory logics </strong><br>
	Y Zhu*, G Xu*, <strong>YT Yang*</strong>, Z Xu, X Chen, B Shi, D Xie, <u>ZJ Lu</u>, <u>P Wang</u>.
	(2019) <strong><em>Nucleic Acids Research</em></strong> 47(D1):D203-D211.<br>
	<a href="https://www.researchgate.net/publication/330213005_POSTAR2_deciphering_the_post-transcriptional_regulatory_logics"> Full Text </a>
	<br><br>	
	
<center><h3><font color="#000000"><strong>-- 2018 --</strong></font></h3></center>
	<strong> Comprehensive functional genomic resource and integrative model for the human brain </strong><br>
	D Wang*, S Liu*, J Warrell*, H Won*, X Shi*, FCP Navarro*, D Clarke*, M Gu*, P Emani*, <strong>YT Yang</strong>, M Xu, MJ Gandal, S Lou, J Zhang, JJ Park, C Yan, SK Rhie, K Manakongtreecheep, H Zhou, A Nathan, M Peters, E Mattei, D Fitzgerald, T Brunetti, J Moore, Y Jiang, K Girdhar, GE Hoffman, S Kalayci, ZH Gümüş, GE Crawford, PsychENCODE Consortium, P Roussos, S Akbarian, AE Jaffe, KP White, Z Weng, N Sestan, <u>DH Geschwind</u>, <u>JA Knowles</u>, <u>MB Gerstein</u>.
	(2018) <strong><em>Science</em></strong> 362(6420):eaat8464.<br>
	<a href="https://www.researchgate.net/publication/329705246_Comprehensive_functional_genomic_resource_and_integrative_model_for_the_human_brain"> Full Text </a>
	<br><br>
	
	<strong> Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder </strong><br>
	<u>MJ Gandal</u>, P Zhang, E Hadjimichael, RL Walker, C Chen, S Liu, H Won, H van Bakel, M Varghese, Y Wang, AW Shieh, J Haney, S Parhami, J Belmont, M Kim, PM Losada, Z Khan, J Mleczko, Y Xia, R Dai, D Wang, <strong>YT Yang</strong>, M Xu, K Fish, PR Hof, J Warrell, D Fitzgerald, K White, AE Jaffe, PsychENCODE Consortium, MA Peters, M Gerstein, <u>C Liu</u>, <u>LM Iakoucheva</u>, <u>D Pinto</u>, <u>DH Geschwind</u>.
	(2018) <strong><em>Science</em></strong> 362(6420):eaat8127.<br>
	<a href="https://www.researchgate.net/publication/329648373_Transcriptome-wide_isoform-level_dysregulation_in_ASD_schizophrenia_and_bipolar_disorder"> Full Text </a>
	<br><br>
	
	<strong> Investigation of RNA-RNA interactions using the RISE database </strong><br>
	Y Ju*, J Gong*, <strong><u>YT Yang</u></strong>, <u>QC Zhang</u>.
	(2018) <strong><em>Current Protocols in Bioinformatics</em></strong> 64(1):e58.<br>
	<a href="https://www.researchgate.net/publication/327977102_Investigation_of_RNA-RNA_interactions_using_the_RISE_database"> Full Text </a>
	<br><br>
	
	<strong> RISE: a database of RNA Interactome from Sequencing Experiments </strong><br>
	J Gong*, D Shao*, K Xu, Z Lu, ZJ Lu, <strong><u>YT Yang</u></strong>, <u>QC Zhang</u>.
	(2018) <strong><em>Nucleic Acids Research</em></strong> 46(D1):D194-D201.<br>
	<a href="https://www.researchgate.net/publication/320224534_RISE_A_database_of_RNA_interactome_from_sequencing_experiments"> Full Text </a>
	<br><br>	
	
<center><h3><font color="#000000"><strong>-- 2017 --</strong></font></h3></center>
	<strong> Identification of high-confidence RNA regulatory elements by combinatorial classification of RNA-protein binding sites </strong><br>
	YE Li*, M Xiao*, B Shi*, <strong>YT Yang</strong>, D Wang, F Wang, M Marcia, <u>ZJ Lu</u>.
	(2017) <strong><em>Genome Biology</em></strong> 18(1):169.<br>
	<a href="https://www.researchgate.net/publication/328823193_Identification_of_high-confidence_RNA_regulatory_elements_by_combinatorial_classification_of_RNA-protein_binding_sites"> Full Text </a>
	<br><br>
	
	<strong> Recurrently deregulated lncRNAs in hepatocellular carcinoma </strong><br>
	Y Yang*, <u>L Chen</u>*, J Gu*, H Zhang*, J Yuan, Q Lian, G Lv, S Wang, Y Wu, <strong>YT Yang</strong>, D Wang, Y Liu, J Tang, G Luo, Y Li, L Hu, X Sun, D Wang, M Guo, Q Xi, J Xi, <u>H Wang</u>, <u>MQ Zhang</u>, <u>ZJ Lu</u>.
	(2017) <strong><em>Nature Communications</em></strong> 8:14421.<br>
	<a href="https://www.researchgate.net/publication/313660480_Recurrently_deregulated_lncRNAs_in_hepatocellular_carcinoma"> Full Text </a>
	<br><br>
	
	<strong> Large-scale mapping of mammalian transcriptomes identifies conserved genes associated with different cell differentiation states </strong><br>
	Y Yang*, <strong>YT Yang*</strong>, J Yuan, <u>ZJ Lu</u>, <u>JJ Li</u>.
	(2017) <strong><em>Nucleic Acids Research</em></strong> 45(4):1657-1672.<br>
	<a href="https://www.researchgate.net/publication/311649862_Large-scale_mapping_of_mammalian_transcriptomes_identifies_conserved_genes_associated_with_different_cell_states"> Full Text </a>
	<br><br>
	
	<strong> Comprehensive analysis of long non-coding RNAs highlights their spatio-temporal expression patterns and evolutional conservation in Sus scrofa </strong><br>
	<u>Z Tang</u>*, Y Wu*, Y Yang*, <strong>YT Yang</strong>, Z Wang, J Yuan, Y Yang, C Hua, X Fan, G Niu, Y Zhang, <u>ZJ Lu</u>, <u>K Li</u>.
	(2017) <strong><em>Scientific Reports</em></strong> 7:43166.<br>
	<a href="https://www.researchgate.net/publication/313988739_Comprehensive_analysis_of_long_non-coding_RNAs_highlights_their_spatio-temporal_expression_patterns_and_evolutional_conservation_in_Sus_scrofa"> Full Text </a>
	<br><br>
	
	<strong> POSTAR: a platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins </strong><br>
	B Hu*, <strong>YT Yang*</strong>, Y Huang, Y Zhu, <u>ZJ Lu</u>.
	(2017) <strong><em>Nucleic Acids Research</em></strong> 45(D1):D104-D114.<br>
	<a href="https://www.researchgate.net/publication/308898793_POSTAR_A_platform_for_exploring_post-transcriptional_regulation_coordinated_by_RNA-binding_proteins"> Full Text </a>
	<br><br>	
	
<center><h3><font color="#000000"><strong>-- 2016 --</strong></font></h3></center>
	<strong> CCG: an integrative resource of cancer protein-coding and long noncoding RNA genes </strong><br>
	M Liu*, <strong>YT Yang*</strong>, G Xu, C Tan, <u>ZJ Lu</u>.
	(2016) <strong><em>Discovery Medicine</em></strong> 22(123):351-359.<br>
	<a href="https://www.researchgate.net/publication/309669411_CCG_An_Integrative_Resource_of_Cancer_Protein-coding_Genes_and_Long_Noncoding_RNAs"> Full Text </a>
	<br><br>
	
<center><h3><font color="#000000"><strong>-- 2015 --</strong></font></h3></center>
	<strong> A common set of distinct features that characterize noncoding RNAs across multiple species </strong><br>
	L Hu, C Di, M Kai, <strong>YT Yang</strong>, Y Li, Y Qiu, X Hu, KY Yip, MQ Zhang, <u>ZJ Lu</u>.
	(2015) <strong><em>Nucleic Acids Research</em></strong> 43(1):104-114.<br>
	<a href="https://www.researchgate.net/publication/269455407_A_common_set_of_distinct_features_that_characterize_noncoding_RNAs_across_multiple_species"> Full Text </a>
	<br><br>
	
	<strong> CLIPdb: a CLIP-seq database for protein-RNA interactions </strong><br>
	<strong>YT Yang*</strong>, C Di*, B Hu*, M Zhou, Y Liu, N Song, Y Li, J Umetsu, <u>ZJ Lu</u>.
	(2015) <strong><em>BMC Genomics</em></strong> 16(1):51-58.<br>
	<a href="https://www.researchgate.net/publication/271863207_CLIPdb_a_CLIP-seq_database_for_protein-RNA_interactions"> Full Text </a>
	<br><br>	
	
<center><h3><font color="#000000"><strong>-- 2014 --</strong></font></h3></center>
	<strong> Global signatures of protein binding on structured RNAs in Saccharomyces cerevisiae </strong><br>
	<strong>Y-C Yang</strong>, J Umetsu, <u>ZJ Lu</u>.
	(2014) <strong><em>Science China Life Sciences</em></strong> (Tsinghua Special Issue) 57(1):22-35.<br>
	<a href="https://www.researchgate.net/publication/259450822_Global_signatures_of_protein_binding_on_structured_RNAs_in_Saccharomyces_cerevisiae"> Full Text </a>
	<br><br>	
	
<center><h3><font color="#000000"><strong>-- 2013 --</strong></font></h3></center>
	<strong> Genomic study of polyhydroxyalkanoates producing Aeromonas hydrophila 4AK4 </strong><br>
	X Gao, J Jian, W-J Li, <strong>Y-C Yang</strong>, X-W Shen, Z-R Sun, Q Wu, <u>G-Q Chen</u>.
	(2013) <strong><em>Applied Microbiology and Biotechnology</em></strong> 97(20):9099-9109.<br>
	<a href="https://www.researchgate.net/publication/256378314_Genomic_study_of_polyhydroxyalkanoates_producing_Aeromonas_hydrophila_4AK4"> Full Text </a>
	<br><br>
	
	<strong> Pervasive and dynamic protein binding sites of the mRNA transcriptome in Saccharomyces cerevisiae </strong><br>
	MA Freeberg*, T Han*, JJ Moresco, A Kong, <strong>Y-C Yang</strong>, ZJ Lu, JR Yates, <u>JK Kim</u>.
	(2013) <strong><em>Genome Biology</em></strong> 14(2):R13.<br>
	<a href="https://www.researchgate.net/publication/328812931_Pervasive_and_dynamic_protein_binding_sites_of_the_mRNA_transcriptome_in_Saccharomyces_cerevisiae"> Full Text </a>
	<br><br>

	
<h2><font color="#0371BB">Tools</font></h2>
	
<strong><a href="http://lulab.life.tsinghua.edu.cn/postar3/index.html">POSTAR</a></strong>
<ul>
	<li>A plantform for exploring post-transcriptional regulation coordinated by RNA-binding proteins (Maintained by Prof. Zhi Lu's Lab @ Tsinghua)</li>
	<li>Citation: Zhao et al. POSTAR3: an updated platform for exploring post-transcriptional regulation coordinated by RNA-binding proteins. (2022) <em>Nucleic Acids Research</em> 50:D287-D294.</li>
</ul>
	
<br><br>

<strong><a href="http://rise.life.tsinghua.edu.cn/">RISE</a></strong>
<ul>
	<li>A database of RNA-RNA interactome from high-throughput sequencing experiments (Maintained by Prof. Qiangfeng Zhang's Lab @ Tsinghua)</li>
	<li>Citation: Gong et al. RISE: a database of RNA Interactome from Sequencing Experiments. (2018) <em>Nucleic Acids Research</em> 46:D194-D201.</li>
</ul>

<br><br>


<h2><font color="#0371BB">Data Resource</font></h2>

<strong>Supplementary data to the EN-TEx project</strong>
<ul>
	<li><a href="http://entex.gersteinlab.org/">Download link</a></li>
	<li>Citation: Rozowsky et al. Multi-tissue integrative analysis of personal epigenomes. <em>bioRxiv</em>.</li>
</ul>
	
<br><br>

<strong>Estimates of gene expression (FPKM) in various cell and tissue types from human, chimpanzee, bonobo and mouse</strong>
<ul>
	<li><a href="https://github.com/yucheng-yang/MammTROM">Download link</a></li>
	<li>Citation: Yang et al. Large-scale mapping of mammalian transcriptomes identifies conserved genes associated with different cell states. (2017) <em>Nucleic Acids Research</em> 45:1657–1672.</li>
</ul>

<br><br>


<h2><font color="#0371BB">Teaching</font></h2>

<strong>Introduction to Computational Genomics</strong> (STBI737010)<br>
2021 Fall<br>
	
	<br>

<strong>Introduction to Bioinformatics</strong> (STBI820002, As Co-Instructor)<br>
2022 Spring, 2021 Spring</a><br>



<!-- <br><br>
<h2><font color="#0371BB">Activities</font></h2>
<figure>
	<img src="./images/MICCAI2020.jpg" border="1" width="600" align="left" style="box-shadow: 4px 4px 8px #888; margin-left:10px; margin-right:10px; margin-top: 10px; margin-bottom: 10px;">
	<figcaption>I presented VULD @ <b>MICCAI2020</b>.</figcaption>
</figure> -->
<div id="footer">
	<div id="footer-text"></div>
</div>
<!-- <div style="text-align:right;">
<b><font color="gray">Home</font></b>&nbsp;&nbsp;|&nbsp; <b><a href="team.html">Team</a></b>&nbsp;&nbsp;|&nbsp; <b><a href="research.html">Research</a></b>&nbsp;&nbsp;|&nbsp; <b><a href="publications.html">Publications</a></b>
</div> -->

<p align="center"><iframe style="background:transparent !important" scrolling="no" frameborder="0" allowtransparency="true" width="90" height="90" src="./datafiles/a.html"></iframe><script type="text/javascript" src="./datafiles/5.js" async="async"></script></p>

<div id="feedly-mini" title="feedly Mini tookit"></div><div class="jvectormap-tip"></div></div></body></html>
